packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -1,7 +1,9 @@
 import { ISliderStyleProps, ISliderStyles } from './Slider.types';
-import { getGlobalClassNames, HighContrastSelector, AnimationVariables, getFocusStyle } from '../../Styling';
+import { getGlobalClassNames, HighContrastSelector, AnimationVariables, getFocusStyle, IRawStyle } from '../../Styling';
 import { getRTL } from '@uifabric/utilities';
 
+const SliderSpacing = 13;
```

Comment:
Suggestion: Maybe something like `tickLabelSpacing` (?)

Reply:
ok changed the name to that